packages/cubejs-schema-compiler/src/adapter/PreAggregations.js

```diff
@@ -362,82 +363,175 @@ export class PreAggregations {
     }
   }
 
+  /**
+   * Returns function to determine whether pre-aggregation can be used or not
+   * for specified query, or its value for `refs` if specified.
+   * @param {Object} transformedQuery transformed query
+   * @param {Object?} refs pre-aggs reference
+   * @returns {function(preagg: Object): boolean}
+   */
   static canUsePreAggregationForTransformedQueryFn(transformedQuery, refs) {
```

Comment:
A bit unclear what a "transformedQuery" is. Could this function be called `canUsePreAggregationFn`? I also see above `PreAggregations.transformQueryToCanUseForm(query)`. Sounds like `canUseForm` is a specific query format. Would be awesome if it were explicitly documented in a docstring. For example:

```
/**
 * Returns function to determine whether pre-aggregation can be used or not
 * for specified query, or its value for `refs` if specified.
 * @param {Object} canUseQuery query in canUseForm, i.e. <has property A and B>
 * @param {Object?} refs pre-aggs reference
 * @returns {function(preagg: Object): boolean}
 */
static canUsePreAggregationFn(canUseQuery, refs) {
    ...
}
```

Reply:
By the way, `transformedQuery` is a transformed for the processing query and some parts of it differ from what we send from the client.